charts/akeyless-secure-remote-access/templates/statefulSet.yaml

```diff
@@ -104,6 +104,14 @@ spec:
           - name: USERNAME_SUB_CLAIM
             value: {{ .Values.usernameSubClaim }}
 {{- end }}
+{{- if .Values.ztbConfig.usernameSubClaim }}
```

Comment:
I don't think SSH has this parameter

Reply:
This is the new usernameSubClaim parameter under sshConfig section.